code/modules/mob/living/carbon/carbon.dm

```diff
@@ -650,16 +650,23 @@
 		if(health<= HEALTH_THRESHOLD_DEAD || !getorgan(/obj/item/organ/brain))
 			death()
 			return
-		if(paralysis || sleeping || getOxyLoss() > 50 || (status_flags & FAKEDEATH) || health <= HEALTH_THRESHOLD_CRIT)
+		if(paralysis || sleeping || getOxyLoss() > 50 || (status_flags & FAKEDEATH) || health <= HEALTH_THRESHOLD_DEEPCRIT)
 			if(stat == CONSCIOUS)
 				stat = UNCONSCIOUS
 				blind_eyes(1)
 				update_canmove()
+		else if(health <= HEALTH_THRESHOLD_CRIT)
+			status_flags |= NEARCRIT

```

Comment:
Why do you keep setting this flag? Just set it once and clear it once he is out of `HEALTH_THRESHOLD_CRIT`